packages/flutter/test/material/tooltip_test.dart

```diff
@@ -912,6 +912,105 @@ void main() {
     expect(find.text(tooltipText), findsNothing);
   });
 
+  testWidgets('Tooltip text is also when hoverable', (WidgetTester tester) async {
```

Comment:
I think there's a typo in this description, but I'm not sure what you meant to say...